include/z64skybox.h

```diff
@@ -14,45 +14,46 @@ struct GameState;
 struct PlayState;
 
 typedef enum SkyboxId {
-    /* 0 */ SKYBOX_NONE,
-    /* 1 */ SKYBOX_NORMAL_SKY,
-    /* 2 */ SKYBOX_2,
-    /* 3 */ SKYBOX_3,
-    /* 5 */ SKYBOX_CUTSCENE_MAP = 5
+    SKYBOX_NONE,
+    SKYBOX_NORMAL_SKY,
+    SKYBOX_2,
+    SKYBOX_3,
+    SKYBOX_CUTSCENE_MAP = 5
```

Comment:
Please bring back those comments, they indicate the value of each enum value and we need them